src/casync-http.c

```diff
@@ -453,6 +455,11 @@ static int run(int argc, char *argv[]) {
                 const char *store_url;
                 CaChunkID id;
 
+                if (quit) {
+                        log_info("Got exit signal, quitting.");
```

Comment:
Please insert an 'r = 0;' before the goto just to set the exit value as SUCCESS.

Reply:
Done, thanks for pointing that out.